web/server.js

```diff
@@ -129,27 +119,25 @@ module.exports = function(options) {
     return false;
   }
 
-  server.register({
-    register: require('crumb'),
+  await server.register({
+    plugin: require('crumb'),
     options: {
       restful: true,
       skip: !options.enableCSRF ? skipCSRF : undefined,
       cookieOptions: {
-        isSecure: options.secureCookies
+        isSecure: options.secureCookies,
+        isSameSite: false,
```

Comment:
disables the `SameSite` header (which used to be the default behaviour before) which apparently if enabled, does not allow access to the header on the client (Login might be using it?)